### ISSUE TITLE
"bluescreen"

### ISSUE THREAD

**Issue by faelb** [ramboxapp/community-edition#2593]
when logging into my rambox account to get my accounts on other devices, a unlimited bluescreen with scrollbar appears - no way to interact with the software anymore - I tried restart and reload - nothing works.

This happened on windows & Ubuntu

	
	-
	> Rambox 0.7.3
	> Electron 7.1.1
	> win32 x64 10.0.18362
![Anmerkung 2020-02-21 103059](https://user-images.githubusercontent.com/49062695/75021802-47d19800-5495-11ea-8d3a-1c2b07c8ad7f.png)

**Comment by fvulich** [comment]
Does it happen in other versions or it start happen with v0.7.3? Could you try with an older version? You can download them from [HERE](https://github.com/ramboxapp/community-edition/releases)

**Comment by faelb** [comment]
only tried it with this Version - but for each OS-Version (Windows and Linux) it worked at last after some time ( like restarting the PC  and the app several times, waiting for some days : P)

**Comment by Nullmaruzero** [comment]
Same problem here.

	
	-
	> Rambox 0.7.4
	> Electron 7.1.12
	> win32 x64 10.0.18363

I've downloaded Rambox (first time user), configured everything, set up a Rambox account, verified, logged in, pushed the configuration. Did some minor changes, pushed again. Added another instance of Protonmail and that's where the UI stopped reacting, after creating the instance the pop-up window did not disappear but I could see a new entry has been added (on the list on the main tab). So I have clicked again, it has added another one (duplicate).
I thought it is just Electron being Electron, so I have closed Rambox, restarded and it came up fresh, with everything gone. I have logged in to my account, for a split of a second I could see the tabs populating aaaaand the blue screen appeared with constantly changing size (guessing by the sizes of the scrollbars changing).
I've restarted the app again, fresh vanilla again, this time I have cleared the cache and local storage via Help -> Tools. Logged in again, same problem.

I have then reinstalled the app (run the installer over again, assuming some file corruption). It worked, after starting Rambox up I was already logged in, my tabs were there but I was logged out out of each of them (expected). So I have logged in again to each of the services, tried to push the configuration but a synch error kept appearing.
> Error Appeared when trying to backup your configuration.

Taking a quick look into the dev tools' console shows Status 400 errors.
```
Failed to load resource: the server responded with a status of 400 ()
```

So I have tried to log out, and the blue screen problem appeared again.
